src/WebUI/src/pages/characters/[id]/inventory.vue

```diff
@@ -55,6 +55,10 @@ const changeEquippedItems = async (items: EquippedItemId[]) => {
 };
 
 const onSellUserItem = async (itemId: number) => {
+  // reset filter
```

Comment:
Can you make your comment about why you are doing it rather than what you are doing. It's not clear why it's needed.